package.json

```diff
@@ -2,9 +2,13 @@
   "name": "buble",
   "version": "0.18.0",
   "description": "The blazing fast, batteries-included ES2015 compiler",
-  "main": "dist/buble.umd.js",
+  "main": "dist/buble.cjs.js",
```

Comment:
main/module targets node (actually module wont probably be picked up by node automatically, but is kept for parity)

browser is targeting bundlers (we overridy both cjs and es targets, so bundler can pick up appropriate file)

unpkg has what was previously a main - which is umd bundle, so "universal" - but imho we do not need to have umd wrapper for main entry, so ive switched to using cjs there